theseus/labs/lie/functional/utils.py

```diff
@@ -22,3 +22,7 @@ def get_cls_module(cls):
 
 def shape_err_msg(data_type: str, expected_shape: str, tensor_shape: torch.Size) -> str:
     return f"{data_type} must have shape {expected_shape} but got shape {tensor_shape}."
+
+
+def fill_dims(tensor: torch.Tensor, dim: int):
```

Comment:
Is this still used? I didn't see any calls to this function.

Reply:
Not used. But would like to keep it now to see if it will be used in the future.